packages/app/src/components/catalog/shared/OverviewWrapper.tsx

```diff
@@ -15,13 +13,6 @@ const OverviewWrapper = ({ children }: { children?: React.ReactNode }) => (
       <Grid item xs={12}>
         <InfoCard />
       </Grid>
-      <Grid item xs={12}>
-        <EntitySwitch>
-          <EntitySwitch.Case if={hasLinks}>
-            <EntityLinksCard cols={1} />
-          </EntitySwitch.Case>
-        </EntitySwitch>
-      </Grid>
```

Comment:
Is it possible to replace this with
 
```typescript
const { entity } = useEntity();

<EntitySwitch>
...
      <Typography variant="body1">
        Self service, quick access and links to learn more about this {entity.kind}
      </Typography>
...
</EntitySwitch>
```
so that every entity doesn't have to add the `LinkTiles` separatly and it only added to the `OverviewWrapper`?

Reply:
This is not possible, since we don't maintain the same link placement across all views. In `Component` and `Resource` we display it as a second row on the left. On `System`, `Domain` we display it as the topmost action on the left column.